src/components/TemplateLibrary/index.js

```diff
@@ -215,7 +207,7 @@ const LibraryComponent = () => {
                   }
                 }
               }
-              if(!identifierExists){
+              if (!identifierExists) {
```

Comment:
There were some linting errors that were auto fixed by calling `npm run lint:fix` I may revert these changes as they are not applicable for this PR